File: src/gatling/scala/org/openapitools/client/model/GpsDTO.scala

package org.openapitools.client.model


case class GpsDTO (
    /* Широта. */
    _latitude: Number,
    /* Долгота. */
    _longitude: Number
)
object GpsDTO {
    def toStringBody(var_latitude: Object, var_longitude: Object) =
        s"""
        | {
        | "latitude":$var_latitude,"longitude":$var_longitude
        | }
        """.stripMargin
}

File: API/Client/GpsDTO.js
goog.provide('API.Client.GpsDTO');

/**
 * GPS-координаты широты и долготы. 
 * @record
 */
API.Client.GpsDTO = function() {}

/**
 * Широта.
 * @type {!number}
 * @export
 */
API.Client.GpsDTO.prototype.latitude;

/**
 * Долгота.
 * @type {!number}
 * @export
 */
API.Client.GpsDTO.prototype.longitude;

